### PR TITLE
Fixed typos in docs, comments and error message text.

### PR DESCRIPTION
**Pull request by thnkslprpt**
**Describe the contribution**
Fixed various minor text typos in README docs, comments and a few debug/error message strings.
  
**Testing performed**
None (non-executable code).

**Expected behavior changes**
None. Code execution is unchanged/unaffected.

**System(s) tested on**
n/a

**Additional context**
n/a

**Code contributions**
n/a

### CODE REVIEW

**Review by astrogeco — CHANGES_REQUESTED** [review]
This is very thorough, thank you! Did you use a spellcheck script? 

My only request is that you split commits with one only dealing with documentation typos (.txt and .md files) another commit that addresses "comment" typos and one last commit that addresses actual source code typos (like the print statement for seMinitialize).

**Comment by thnkslprpt** [comment]
Yeah no worries Gerardo - I thought it might be better to split them but wasn't sure.
I'll split up all of the remaining pull requests for the submodules that haven't been merged in yet.

Re. spellcheck script? No - maybe if I knew how to do that I would have haha!
Most were found by CTRL+F'ing typos that I noticed and finding the same typo littered throughout the the various submodules. For some of the more 'wordy' files I also pasted them into Google Docs and checked what it found.

File: README.md [hunk 1]
```diff
@@ -134,14 +134,14 @@ See <https://github.com/nasa/osal/pull/1050> and <https://github.com/nasa/cFS/pu
...
 ### Development Build: v5.1.0-rc1+dev417
 
-- Fixes infinite loop  in `UtPrintx()`. Adds the data's memory address to output. Note, UtPrintf displays the the file/line of the `UtPrintx` function, **not the actual test location**; it is better to call `UT_BSP_DoText` directly.
+- Fixes infinite loop  in `UtPrintx()`. Adds the data's memory address to output. Note, UtPrintf displays the file/line of the `UtPrintx` function, **not the actual test location**; it is better to call `UT_BSP_DoText` directly.
 - Adds `OS_SocketShutdown()` API wrapper around BSD's socket shutdown() API. This allows a data transfer of a stream socket to be gracefully shut down prior to socket closure.
 - See osal <https://github.com/nasa/osal/pull/979> and <https://github.com/nasa/cFS/pull/252>
 
 ### Development Build: v5.1.0-rc1+dev411
 
 - [docs] Clarifies that that zero will be returned on EOF condition in the API documentation for OS_read/write/TimedRead/TimedWrite. In the case of the timed API calls, the `OS_ERR_TIMEOUT` status code will be returned if the timeout expired without the handle becoming readable/writable during that time.
```
Comment (ArielSAdamsNASA) [inline README.md:143]: Extra that in `[docs] Clarifies that that zero will be returned on EOF condition in the API documentation for OS_read/write/TimedRead/TimedWrite.`

**Review by ArielSAdamsNASA — CHANGES_REQUESTED** [review]

File: src/bsp/pc-rtems/src/bsp_start.c [hunk 2]
```diff
@@ -385,7 +385,7 @@ rtems_task Init(rtems_task_argument ignored)
 /* configuration information */
```
Comment (ArielSAdamsNASA) [inline src/bsp/pc-rtems/src/bsp_start.c:385]: `* Batch mode is intended for non-interative execution.` Misspelled on line 104

**Review by ArielSAdamsNASA — CHANGES_REQUESTED** [review]

**Comment by thnkslprpt** [comment]
> This is very thorough, thank you!

Haha - re. 'thorough' - yeah. I suppose I thought... if I'm going to do it, may as well do it properly. (although I did leave in a few hundred _very_ minor mistakes like its/it's etc. which are below even my threshold for worth correcting ;-)

File: src/bsp/pc-rtems/src/bsp_start.c (hunk 2 above)
Comment (thnkslprpt) [reply src/bsp/pc-rtems/src/bsp_start.c:385]: It's supposed to be "non-interactive", correct? (not non-iterative)

File: README.md (hunk 1 above)
Comment (thnkslprpt) [reply README.md:143]: Cheers Ariel. I'll add this change.

File: src/tests/file-api-test/README.txt [hunk 3]
```diff
@@ -17,22 +17,22 @@ It also tests removing files that are not on the drive.
```
Comment (ArielSAdamsNASA) [inline src/tests/file-api-test/README.txt:17]: `(i.e. you cannot open a file for reading/writing without being able to create the file, and you can testing reading/writing without being able to open a file)` change testing to test on line 7.

**Review by ArielSAdamsNASA — CHANGES_REQUESTED** [review]

File: src/bsp/pc-rtems/src/bsp_start.c (hunk 2 above)
Comment (ArielSAdamsNASA) [reply src/bsp/pc-rtems/src/bsp_start.c:385]: @jphickey @skliper Thoughts?

Comment (jphickey) [reply src/bsp/pc-rtems/src/bsp_start.c:385]: yes, it should be "non-interactive"

Comment (thnkslprpt) [reply src/bsp/pc-rtems/src/bsp_start.c:385]: Thanks Joe.

**Comment by thnkslprpt** [comment]
@astrogeco - The commit has been split into comments/docs/code as requested, and the additional typos that @ArielSAdamsNASA noticed have been incorporated.

**Review by ArielSAdamsNASA — APPROVED** [review]

**Review by astrogeco — APPROVED** [review]

**Comment by astrogeco** [comment]
Tested in https://github.com/astrogeco/osal/commit/2a2defbb6b5ecc9a0f6d5af96ff73f88edc27327

Merged in 2a2defbb6b5ecc9a0f6d5af96ff73f88edc27327

**Pull request merged**